compiler/env/Region.hpp

```diff
@@ -152,6 +152,13 @@ class Region
       } _initialSegmentArea;
 };
 
+
+static void reset(TR::Region& targetRegion, TR::Region& prototypeRegion)
+   {
+   targetRegion.~Region(); 
+   new (&targetRegion) TR::Region(prototypeRegion);
+   }
```

Comment:
This is supposed to be a static member function, so it needs to be inside the definition of `class TR::Region`.